julia/core.py

```diff
@@ -286,7 +289,11 @@ def __init__(self, init_julia=True, jl_init_path=None):
                     "Install PyCall by running the following line:\n"
                     """\tjulia -e 'Pkg.add("PyCall"); Pkg.update()'\n""")
             try:
-                self.call('pyinitialize(C_NULL)')
+                if os.name == "nt":
+                    # on windows C_NULL doesn't mean anything
+                    self.call('pyinitialize()')
```

Comment:
Well, with `C_NULL` on winpython, I have this error:
`symbol could not be found Py_GetVersion (-1): no error`


![julia_c_null](https://cloud.githubusercontent.com/assets/4312421/6254166/dfd76d32-b7a6-11e4-8d48-c13de0e1bf6e.GIF)


Wuthout `C_NULL`, it works